PathTracer/kernel.cu

```diff
@@ -11,6 +11,43 @@
 //Define BVH_DEBUG to zero to output only what the BVH looks like
 #define BVH_DEBUG 0
 
+#if BVH_DEBUG
```

Comment:
I moved this to the top so that it's cleaner and function only exists if BVH_DEBUG is set.